src/types.h

```diff
@@ -152,3 +162,9 @@ struct parsed_transaction {
         __typeof__(b) ____b_ = (b);                                                                                    \
         ____a_ < ____b_ ? ____a_ : ____b_;                                                                             \
     })
+
+typedef struct {
+	bool initialized;
+	bool show_testnet;
+	char network_prompt[10];
```

Comment:
I'm surprised this needs to go in nvram. Can't you get the string from the value of show_testnet?

Reply:
Yes, but so far as I know we use 64 bytes of nvram either way and i am more confident that the menu system will operate properly with them there.